src/main/kotlin/com/wafflestudio/team2server/storage/service/StorageService.kt

```diff
@@ -0,0 +1,68 @@
+package com.wafflestudio.team2server.storage.service
+
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.DeleteObjectRequest
+import com.amazonaws.services.s3.model.ObjectMetadata
+import com.amazonaws.services.s3.model.PutObjectRequest
+import com.wafflestudio.team2server.common.error.BaniException
+import com.wafflestudio.team2server.common.error.ErrorType
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.web.multipart.MultipartFile
+import java.util.*
+
+
+@Service
+class StorageService(
+	@Autowired
+	private val amazonS3: AmazonS3
+) {
+	@Value("\${cloud.aws.s3.bucket}")
+	private val bucket: String? = null
+
+
+	fun uploadFile(multipartFiles: List<MultipartFile>): List<String> {
+		return multipartFiles.map { file ->
+			val fileName = createFileName(file.originalFilename ?: "")
+			val objectMetadata = ObjectMetadata()
+			objectMetadata.contentLength = file.size
+			objectMetadata.contentType = file.contentType
+			amazonS3.putObject(
+				PutObjectRequest(bucket, fileName, file.inputStream, objectMetadata)
+			)
+			amazonS3.getUrl(bucket, fileName).toString()
+		}
+	}
+	/*
+	fun uploadFile(multipartFile: MultipartFile): String {
+		val fileName = createFileName(multipartFile.originalFilename ?: "")
+		val objectMetadata = ObjectMetadata()
+		objectMetadata.contentLength = multipartFile.size
+		objectMetadata.contentType = multipartFile.contentType
+		amazonS3.putObject(
+			PutObjectRequest(bucket, fileName, multipartFile.inputStream, objectMetadata)
+		)
+		return amazonS3.getUrl(bucket, fileName).toString()
+	}*/
+
+	fun getFileExtension(fileName: String): String {
+		val ext = try {
+			fileName.substring(fileName.lastIndexOf("."))
+		} catch (e: StringIndexOutOfBoundsException) {
+			throw BaniException(ErrorType.BAD_FILE_FORMAT)
+		}
+		if (ext == ".jpg" || ext == ".jpeg" || ext == ".png") { // 업로드 가능 확장자
+			return ext
+		}
+		throw BaniException(ErrorType.BAD_FILE_FORMAT)
+	}
```

Comment:
마지막 줄에 예외를 던지는 것보다 먼저 던지는 것이 좋을 것 같아요. 다음과 같이 변경하면 더 깔끔할 듯요.
```suggestion
	fun getFileExtension(fileName: String): String {
		val ext = filename.substringAfterLast(".", "")
		if (ext != ".jpg" && ext != ".jpeg" && ext != ".png") { // 업로드 가능 확장자
			throw BaniException(ErrorType.BAD_FILE_FORMAT)
		}
		return ext
	}
```